### DISCUSSION TITLE
https://docs.okd.io/ is currently responding with "Application is not available"

### DISCUSSION THREAD

**Discussion by markandrewj** [okd-project/okd#939] [Q&A] [answered]
https://docs.okd.io/ is currently responding with "Application is not available", it's been like this for more then a day now.

![2021-10-20 14_13_02-Window](https://user-images.githubusercontent.com/142597/138165392-a73121c2-9580-4aea-992b-7eb75d47fdb5.png)

**Comment by vrutkovs** [comment, marked as answer]
https://github.com/openshift/okd/discussions/934#discussioncomment-1507879
